src/runtime/runtime.cs

```diff
@@ -1985,7 +1923,7 @@ static Delegates()
                 }
                 catch (MissingMethodException e)
                 {
-                    throw new NotSupportedException(Util.MinimalPythonVersionRequired, innerException: e);
+                    throw new NotSupportedException(Util.MinimalPythonVersionRequired35, innerException: e);
```

Comment:
The minimal version we support right now is 3.6.

Reply:
Reverted constant name, edited constant text instead